akka-docs/rst/java/code/docs/stream/ActorPublisherDocTest.java

```diff
@@ -21,100 +22,100 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ActorPublisherDocTest {
+public class ActorPublisherDocTest extends AbstractJavaTest {
 
   static ActorSystem system;
-
+  static Materializer mat;
 
   @BeforeClass
   public static void setup() {
     system = ActorSystem.create("ActorPublisherDocTest");
+    mat = ActorMaterializer.create(system);
   }
 
   @AfterClass
   public static void tearDown() {
     JavaTestKit.shutdownActorSystem(system);
     system = null;
+    mat = null;
   }
 
-  final Materializer mat = ActorMaterializer.create(system);
```

Comment:
This would somehow lead to NPE:s when the test cases was run, can't recall if it was because `system` was `null` at this point of the `Materializer` never got created.